recipes/sra-tools/build.sh

```diff
@@ -1,56 +1,85 @@
-#!/bin/bash
+#!/bin/bash -e
 
-export CFLAGS="-I$PREFIX/include"
-export LDFLAGS="-L$PREFIX/lib"
-export CPATH=${PREFIX}/include
+# Build uses hard-coded references to gcc/g++/etc. and does not properly honor all *FLAGS.
+make_wrapper() {
+    [ -n "${2:+x}" ] || return 0
+    local exe
+    exe="${BUILD_PREFIX}/bin/${3}"
+    touch "${exe}"
+    chmod +x "${exe}"
+    cat > "${exe}" <<EOF
+#! /bin/sh
+exec ${2} ${1} ${CPPFLAGS} ${LDFLAGS} "\${@}"
+EOF
+}
+make_wrapper "${CFLAGS}" "${CC}" cc
+make_wrapper "${CFLAGS}" "${GCC}" gcc
+make_wrapper "${CFLAGS}" "${CLANG}" clang
+make_wrapper "${CXXFLAGS}" "${CXX}" c++
+make_wrapper "${CXXFLAGS}" "${GXX}" g++
+make_wrapper "${CXXFLAGS}" "${CLANGXX}" clang++
+ln -s "${AR}" "${BUILD_PREFIX}/bin/ar"
+ln -s "${LD}" "${BUILD_PREFIX}/bin/ld"
 
-NCBI_OUTDIR=$SRC_DIR/ncbi-outdir
-
-if [[ $OSTYPE == darwin* ]]; then
-     export CFLAGS="${CFLAGS} -i sysroot ${CONDA_BUILD_SYSROOT}"
-     export LDFLAGS="${LDFLAGS} -headerpad_max_install_names"
-fi
-
-export PATH=$BUILD_PREFIX/bin:$PATH
 
-ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-gcc $BUILD_PREFIX/bin/gcc
-ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-c++ $BUILD_PREFIX/bin/g++
-ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-cc $BUILD_PREFIX/bin/cc
-ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-c++ $BUILD_PREFIX/bin/c++
-ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-ar $BUILD_PREFIX/bin/ar
-ln -s $BUILD_PREFIX/bin/x86_64-conda_cos6-linux-gnu-ld $BUILD_PREFIX/bin/ld
+NCBI_OUTDIR=$SRC_DIR/ncbi-outdir
 
 
 echo "compiling ncbi-vdb"
 pushd ncbi-vdb
+
+{
+cat <<end-of-patch
+--- libs/kns/tls.c
++++ libs/kns/tls.c
+@@ -431,4 +431,5 @@
+         const char * root_ca_paths [] =
+         {
++            "${PREFIX}", /* conda-forge::ca-certificates */
```

Comment:
Shouldn't this be
```suggestion
+            "${PREFIX}/ssl", /* conda-forge::ca-certificates */
```
?

Reply:
Whoops, yep, even
```suggestion
+            "${PREFIX}/ssl/cacert.pem", /* conda-forge::ca-certificates */
```

Reply:
I'm out for today, so feel free to hack away in case you find anything else :)